pom.xml

```diff
@@ -172,6 +172,28 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.ec4j.maven</groupId>
+                <artifactId>editorconfig-maven-plugin</artifactId>
+                <version>0.1.1</version>
+                <executions>
+                    <execution>
+                        <id>check</id>
+                        <phase>verify</phase>
```

Comment:
I'll move this to the `compile` phase if you don't mind. That way, if you run `mvn test`, editorconfig will be run before all the tests.